### DISCUSSION TITLE
qBittorrent

### DISCUSSION THREAD

**Discussion by rollingmoai** [catppuccin/catppuccin#1835] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
No

**Additional context 📷 (optional)**
[Examples of qBittorrent themes](https://github.com/qbittorrent/qBittorrent/wiki/List-of-known-qBittorrent-themes)

**Comment by nekoxuee** [comment]
I've made a port for qbittorrent.
repo: https://github.com/Xurdejl/qbittorrent

**Comment by sgoudham** [comment]
https://github.com/catppuccin/qbittorrent now exists.
